qabelbox/src/main/java/de/qabel/qabelbox/providers/BoxProvider.java

```diff
@@ -599,7 +599,7 @@ private File getFile(CancellationSignal signal, final String documentId)
         final NotificationCompat.Builder mBuilder = new NotificationCompat.Builder(getContext());
         mBuilder.setContentTitle("Downloading " + mDocumentIdParser.getBaseName(documentId))
                 .setContentText("Download in progress")
-                .setSmallIcon(R.drawable.notification_template_icon_bg);
+                .setSmallIcon(R.drawable.qabel_logo);
```

Comment:
please add notification icons in correct size for all resolutions (hdpi...xxxhdpi)